src/app/features/management/user-management-view/edit-user-view/edit-user-view.component.html

```diff
@@ -0,0 +1,41 @@
+<div class="view" [appShowLoadingOverlay]="updatingUser.loadingChange() | async">
+  <div class="heading">
+    <h1 i18n>Update User</h1>
+    <button mat-button type="button" (click)="changePassword()" i18n>
+      Change Password
+    </button>
+  </div>
+  <form (submit)="updateUser()" class="example-form">
+    <mat-form-field class="app-edit-input">
+      <mat-label i18n>Username</mat-label>
+      <input matInput [formControl]="form.controls.username">
+      <mat-error *ngIf="form.controls.username.errors && form.controls.username.errors['required']" i18n>Username is
+        required.
+      </mat-error>
+    </mat-form-field>
+    <mat-form-field class="app-edit-input">
+      <mat-label i18n>First Name</mat-label>
+      <input matInput [formControl]="form.controls.firstName">
+      <mat-error *ngIf="form.controls.firstName.errors && form.controls.firstName.errors['required']" i18n>First name is
+        required.
+      </mat-error>
+    </mat-form-field>
+    <mat-form-field class="app-edit-input">
+      <mat-label i18n>Last Name</mat-label>
+      <input matInput [formControl]="form.controls.lastName">
+      <mat-error *ngIf="form.controls.lastName.errors && form.controls.lastName.errors['required']" i18n>Last name is
+        required.
+      </mat-error>
+    </mat-form-field>
+    <mat-checkbox [formControl]="form.controls.isAdmin">Is Admin</mat-checkbox>
+    <mat-checkbox [formControl]="form.controls.isActive">Is Active</mat-checkbox>
+    <div class="button-row">
+      <button mat-button type="button" (click)="cancel()" i18n>
+        Cancel
+      </button>
+      <button mat-button type="submit" [disabled]="!form.valid || (updatingUser.loadingChange() | async)" i18n>
```

Comment:
Form should be disabled as well while loading.

Suggestion:
Replace in `loader.ts`:
```ts

  /**
   * Number of in progress requests.
   * @private
   */
  private inProgress = 0;

  /**
   * Emits the current loading state.
   */
  loadingChange(): Observable<boolean> {
    return this.loadingChangeSubject.asObservable();
  }

  private emitLoadingChangeIfRequired(): void {
    const lastWasLoading = this.loadingChangeSubject.getValue();
    if (lastWasLoading && this.inProgress === 0) {
      this.loadingChangeSubject.next(false);
    } else if (!lastWasLoading && this.inProgress > 0) {
      this.loadingChangeSubject.next(true);
    }
  }
```
with
```ts
  /**
   * Number of in progress requests.
   * @private
   */
  private inProgress = 0;
  private boundFormGroups: FormGroup[] = [];

  bindFormGroup(fg: FormGroup): void {
    this.boundFormGroups.push(fg);
  }

  /**
   * Emits the current loading state.
   */
  loadingChange(): Observable<boolean> {
    return this.loadingChangeSubject.asObservable();
  }

  private emitLoadingChangeIfRequired(): void {
    const lastWasLoading = this.loadingChangeSubject.getValue();
    if (lastWasLoading && this.inProgress === 0) {
      this.loadingChangeSubject.next(false);
      this.boundFormGroups.forEach(fg => fg.enable());
    } else if (!lastWasLoading && this.inProgress > 0) {
      this.loadingChangeSubject.next(true);
      this.boundFormGroups.forEach(fg => fg.disable());
    }
  }
```
(only draft, assure it works accordingly)

and in component, call `updatingUser.bindFormGroup(this.form)` in constructor or wherever.
Then, remove `.loadingChange() | async` here.